### PR TITLE
Register divyanshudhruv.is-a.dev

### PR DESCRIPTION
**Pull request by divyanshudhruv**
> Please add **`divyanshudhruv.is-a.dev`**

> Site preview - [Click](https://divyanshudhruv.github.io)

### CODE REVIEW

**Comment by divyanshudhruv** [comment]
@CuteDog5695 sorry to ping you. But can you explain, why the workflow is not running? Is anything missing from the json file ?
Please help.

**Comment by CuteDog5695** [comment]
> @CuteDog5695 sorry to ping you. But can you explain, why the workflow is not running? Is anything missing from the json file ? Please help.

github sometimes makes maintainers run workflow (this isnt the first PR to do so), when i have time ill take a look at your PR and if its correct merge it

**Comment by divyanshudhruv** [comment]
> > @CuteDog5695 sorry to ping you. But can you explain, why the workflow is not running? Is anything missing from the json file ? Please help.
> 
> github sometimes makes maintainers run workflow (this isnt the first PR to do so), when i have time ill take a look at your PR and if its correct merge it

Oh, thankyou @CuteDog5695 , please merge is ASAP...

And just a question, I dont want **`divyanshudhruv.is-a.dev`** to change the url to **`divyanshudhruv.github.io`** . I mean I want to display the website in **`divyanshudhruv.is-a.dev`** instead of **`divyanshudhruv.github.io`**. So please do the necessory changes also :)

Thankyou

File: domains/divyanshudhruv.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Its just my portfolio :)",
+    "repo": "https://github.com/divyanshudhruv/divyanshudhruv.github.io",
+    "owner": {
+        "username": "divyanshudhruv",
+        "email": "divyanshudhruv24@gmail.com",
```
Comment (CuteDog5695) [inline domains/divyanshudhruv.json:6]: Well, you forgot to remove the comma here. Please remove that

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by divyanshudhruv** [comment]
@CuteDog5695 Resolved

File: domains/divyanshudhruv.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Its just my portfolio :)",
+    "repo": "https://github.com/divyanshudhruv/divyanshudhruv.github.io",
+    "owner": {
+        "username": "divyanshudhruv"
+        "email": "divyanshudhruv24@gmail.com",
```
Comment (CuteDog5695) [inline domains/divyanshudhruv.json:6]: Add back the comma for username and remove email comma

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by divyanshudhruv** [comment]
Done

**Review by iostpa — APPROVED** [review]
Seems good to me

**Pull request merged**